acados/ocp_nlp/ocp_nlp_sqp.c

```diff
@@ -176,6 +188,12 @@ void ocp_nlp_sqp_opts_initialize_default(void *config_, void *dims_, void *opts_
 
     qp_solver->opts_initialize_default(qp_solver, dims->qp_solver, opts->qp_solver_opts);
 
+    if (config->regularization != NULL)
+    {
+        opts->reg_opts->delta = 1e-4;
```

Comment:
are delta and gamma documented somewhere?

Reply:
In the paper ;)